tools/tscdocgen/main.go

```diff
@@ -546,10 +547,11 @@ func (e *emitter) gatherModules(doc *typeDocNode, parentModule string, k8s bool)
 			name := strings.Trim(child.Name, `"`)
 
 			// If it begins with a "./", simplify it to just the parent name.
-			if isModule && strings.Index(name, "./") == 0 {
+			if isModule {
```

Comment:
I do not understand why the `&& strings.Index(name, "./") == 0` case was here previously.  I haven't encountered any providers that trigger this condition at all yet.  For the changes being made here, it is necessary to do this even for cases where this condition applies, but I have probably broken something for other cases where this did apply previously - I just can't find any to verify and understand what they were trying to accomplish.  Anyone know?

Reply:
I found a few places that use this - and so ended up putting it back as is.